packages/adminrouter/extra/src/README.md

```diff
@@ -462,18 +483,18 @@ README.md in EE Admin Router repository.
 
 ### Authentication
 
-For Admin Router to establish the identity of the subject issuing a
-request, it looks for JWT token in the request. It may be present in following
-places:
+For Admin Router to establish the identity of the subject issuing a request, it
+looks for JWT token in the request. It may be present in the following places:
```

Comment:
"it looks for JWT token in the request"

Especially in the security team we must be rather keen on propagating precise and correct terminology around all the tokens in the system.

First of all, whenever you want to write "JWT token" please write "JSON Web Token" or just "JWT" instead.

Here, however, we can be more specifc and say "DC/OS authentication token" instead.